### DISCUSSION TITLE
Use P2P protocol like IPFS to download data

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#11418] [Ideas]
Have you considered using something like [IPFS](https://ipfs.io/) to speed up map/data downloads? They're quite slow as of now :/
A P2P protocol would allow the community to support with bandwidth.

**Comment by stephan75** [comment]
You are aiming at something like
https://forum.openstreetmap.org/viewforum.php?id=107
or
https://wiki.openstreetmap.org/wiki/Planet.osm#BitTorrent ?

**Reply by ghost** [reply]
Yeah I thought of torrents right after I posted, though I still like the idea of IPFS more (notably deduplication).
